src/components/common/navbar.tsx

```diff
@@ -5,142 +5,258 @@ import Image from 'next/image'
+                                size="large"
+                                aria-label="website navigation"
+                                aria-controls="menu-appbar"
+                                aria-haspopup="true"
+                                onClick={handleOpenNavMenu}
+                            >
+                                <MenuIcon />
+                            </IconButton>
+                            <Menu
+                                id="menu-appbar"
+                                anchorEl={anchorElNav}
+                                anchorOrigin={{
+                                    vertical: "bottom",
+                                    horizontal: "left",
+                                }}
+                                keepMounted
+                                transformOrigin={{
+                                    vertical: "top",
+                                    horizontal: "left",
+                                }}
+                                open={Boolean(anchorElNav)}
+                                onClose={handleCloseNavMenu}
+                            >
+                                <MenuItem onClick={handleCloseNavMenu}>
+                                    <Link href="/stories">
+                                        <Typography textAlign="center" color="primary">
+                                            Stories
+                                        </Typography>
+                                    </Link>
+                                </MenuItem>
+                                <MenuItem onClick={handleCloseNavMenu}>
+                                    <Link href="/paintings/all">
+                                        <Typography textAlign="center" color="primary">
+                                            Paintings
+                                        </Typography>
+                                    </Link>
+                                </MenuItem>
+                                <MenuItem onClick={handleCloseNavMenu}>
+                                    <Link href="/manuscripts">
+                                        <Typography textAlign="center" color="primary">
+                                            Manuscripts
+                                        </Typography>
+                                    </Link>
+                                </MenuItem>
+                                <MenuItem onClick={handleCloseNavMenu}>
+                                    <Link href="/research/incipit-tool">
+                                        <Typography textAlign="center" color="primary">
+                                            Research Tools
+                                        </Typography>
+                                    </Link>
+                                </MenuItem>
+                                <MenuItem onClick={handleCloseNavMenu}>
+                                    <Link href="/about/mission">
```

Comment:
Same here drop down in mobile view isn't working, it just goes to `/about/mission`

Reply:
As discussed on our call there will be a separate ticket for mobile view dropdowns